test.e2e/spec/services.spec.js

```diff
@@ -443,7 +443,8 @@ define(['angular', 'given', 'util'], function(angular, given, util) {
           'deleteById',
           'removeById',
           'count',
-          'prototype$updateAttributes',
+          'prototype$patchAttributes',
+          'updateAttributes',
```

Comment:
Because I see both `updateOrCreate` and `upsert` in the list above, I think both strong-remoting and this SDK support remote method aliases.

I would expect to see `prototype$patchAttributes` and `prototype$updateAttributes` listed for LB 3.0 (and hopefully for 2.x too).